### PR TITLE
Fixes and refactoring per FOGL-1724

### PR DESCRIPTION
**Pull request by amarendra-dianomic**

### CODE REVIEW

**Review by ashish-jabble — CHANGES_REQUESTED** [review]
Please fix reconfigure part

File: python/foglamp/plugins/south/dht11/dht11.py [hunk 1]
```diff
@@ -25,19 +26,28 @@
...
     'plugin': {
-         'description': 'Python module name of the plugin to load',
-         'type': 'string',
-         'default': 'dht11'
+        'description': 'Python module name of the plugin to load',
+        'type': 'string',
+        'default': 'dht11',
+        'readonly': 'true'
+    },
+    'assetName': {
```
Comment (ashish-jabble) [inline python/foglamp/plugins/south/dht11/dht11.py:34]: assetName or assetNamePrefix?

**Review by ashish-jabble — DISMISSED** [review]

File: python/foglamp/plugins/south/dht11/dht11.py (hunk 1 above)
Comment (amarendra-dianomic) [reply python/foglamp/plugins/south/dht11/dht11.py:34]: The rule of thumb for assetNamePrefix and assetName is that we apply assetNamePrefix to those plugins where readings for multiple sensors are recorded separately (e.g. Sensortag, envirophat) and assetName for those where only one reading (even for multiple sensors) is recorded (e.g. dht11).

**Review by Singhal-Vaibhav — CHANGES_REQUESTED** [review]
changing the name of assetName does not reflect the changes with new readings.

**Review by Singhal-Vaibhav — APPROVED** [review]

**Pull request merged**